packages/jest-jasmine2/src/index.ts

```diff
@@ -3,32 +3,30 @@
  *
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
- *
- * @flow
  */
 
-import type {Environment} from 'types/Environment';
-import type {GlobalConfig, ProjectConfig} from 'types/Config';
-import type {SnapshotState} from 'jest-snapshot';
-import type {TestResult} from 'types/TestResult';
-import type Runtime from 'jest-runtime';
-
 import path from 'path';
+import {Environment, Config, TestResult} from '@jest/types';
+import {SnapshotState} from 'jest-snapshot';
+// @ts-ignore TODO Remove ignore when jest-runtime is migrated to TS
```

Comment:
it has been 🙂

Reply:
Yea, I remember it on `jest-circus`. However we deleted these types from there

Reply:
That is why I left it here